### ISSUE TITLE
Duplicate Form

### ISSUE THREAD

**Issue by nupsiefrost** [awsmug/torro-forms#343]
It would be great and helpfull if the button "Duplicate form" would duplicate not only the elements but also the actions (like email notifications and redirections). I am using torro-forms that much and I have to put sooo much time into creating / copypasting all the "action" Informations...

Thank you so much :)

**Comment by mahype** [comment]
Yes this would be a great enhancement! I will add your propose to the list of todos!
